src/python/jobs/NovoplastyJob.py

```diff
@@ -96,19 +98,29 @@ def run(self, fileStore):
             # Select the first read sequence as the seed, and write it
             # into the local temporary directory
             working_dir = fileStore.localTempDir
-            seed_file_path = os.path.join(
-                working_dir, assembler_params["seed_file_name"]
-            )
-            with open(seed_file_path, "w+") as f:
-                with gzip.open(read_one_file_path, "rt") as g:
-                    do_write = False
-                    for line in g:
-                        if line[0] == "@":
-                            do_write = True
-                        if line[0] == "+":
-                            break
-                        if do_write:
-                            f.write(line)
+
+            # Check if seed file exists; I.E., it's been imported
+            if self.seed_file_id:
+                # Read existing seed file
+                seed_file_path = utilities.readGlobalFile(
+                    fileStore, self.seed_file_id, os.path.basename(self.seed_file_id)
+                )
+            else:
+                seed_file_path = os.path.join(
+                    working_dir, assembler_params["seed_file_name"]
+                )
+
+                # Create seed file from first read
```

Comment:
Is it worth mentioning here that the seed file must be in FASTQ format?

Reply:
The seed file default for Novoplasty was FASTA format - I decided to leave it as such because Addgene was already using FASTA for their seed files.

Reply:
I see now that you're referring to the formatting of the first read. I'll update the comments in both places to be more explicit